TechnicalDocuments/NewHiresCheckList.md

```diff
@@ -38,15 +38,46 @@ Make sure you join your Tribe/Squad's Slack channels.
 
 Here's how to get the iOS project up and running.
 
-1. Clone the iOS repository: https://github.com/Babylonpartners/babylon-ios
-1. Set up Git LFS and pull, according to these instructions: https://github.com/Babylonpartners/babylon-ios/wiki/How-to-install-Git-LFS
-1. Globally configure Git to use SSH instead of HTTPS: https://ricostacruz.com/til/github-always-ssh
+1. Globally configure Git to use SSH instead of HTTPS:
      ```
      git config --global url."git@github.com:".insteadOf "https://github.com/"
+     git config --global url."git@github.com:".pushinsteadof "git://github.com/"
+     git config --global url."git@github.com:".pushinsteadof "https://github.com/"
+     ```
+1. Install Homebrew if you don't already have it installed:    
+     ```
+     /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+     ```
+   
+1. Setup Git LFS:
+     ```
+     brew install git-lfs
+     git lfs install
+     ```
+
+1. Clone the iOS repository: https://github.com/Babylonpartners/babylon-ios
```

Comment:
We might want to remind people who use 2FA on their GitHub account (which everybody should use) that they would need to create an SSH key / access token on their machine first to then be able to clone over SSH. Since those instructions are for installation on new Macs...

Reply:
Good point. I didn't run into that as I copied my `~/.ssh/config` directory over from my old Mac 😛